src/js/common/schemaform/FormApp.jsx

```diff
@@ -141,7 +141,7 @@ class FormApp extends React.Component {
       const isPrefill = props.prefillsAvailable.includes(currentForm);
       const saveEnabled = !this.props.formConfig.disableSave;
       if (saveEnabled && (isSaved || isPrefill)) {
-        props.fetchInProgressForm(currentForm, props.formConfig.migrations, isPrefill);
+        props.fetchInProgressForm(currentForm, props.formConfig.migrations, !isSaved && isPrefill);
```

Comment:
I found this when testing. The prefill message was being shown for saved apps when you use the `/resume` route. We know if there's a saved application, so we can tell it to not prefill in that case.

Reply:
Good catch, should we move this logic to line 141 when declaring isPrefill?

Reply:
I think the logic is clearer the way it is now, but I did change the prefill variable name to make it clearer what's being checked.